pyproject.toml

```diff
@@ -46,7 +46,7 @@ version = "0.9.34"
 
 [tool.poetry.dependencies]
 SQLAlchemy = {extras = ["asyncio"], version = ">=1.4.49"}
-beautifulsoup4 = "^4.12.2"
+beautifulsoup4 = {extras = ["html"], optional = true, version = "^4.12.2"}
```

Comment:
don't we still need to add an entry for this under `[tool.poetry.extras]`?

Reply:
Ha whoops, wrong place. Nice catch

Reply:
Alright I have fixed that. Will let you or @logan-markewich merge this one, just to be sure

Reply:
Thanks! It should be good. Did a quick double check and everywhere else we use `bs4` its loaded in lazily with `try` and `except` blocks. So, I think should be optional as well.